viskores/internal/Configure.h.in

```diff
@@ -327,115 +327,109 @@
 #define VISKORES_CUDA_VERSION_MINOR @VISKORES_CUDA_VERSION_MINOR@
 #endif
 
-#if __cplusplus >= 201402L || \
-    ( defined(VISKORES_MSVC) && _MSC_VER >= 1910 ) || \
-    ( defined(VISKORES_ICC)  && defined(__INTEL_CXX11_MODE__) )
-#define VISKORES_HAVE_CXX_14
+#if __cplusplus >= 201703L || (defined(VISKORES_MSVC) && (_MSC_VER >= 1928))
+#define VISKORES_HAVE_CXX_17
 #else
-#error "Viskores requires at least a C++14 compiler"
+#error "Viskores requires at least a C++17 compiler"
 #endif
 
-//Mark if we have enabled logging.
+  //Mark if we have enabled logging.
 #cmakedefine VISKORES_ENABLE_LOGGING
 
-// Define a pair of macros, VISKORES_SWALLOW_SEMICOLON_PRE_BLOCK and
-// VISKORES_SWALLOW_SEMICOLON_POST_BLOCK that can be used around a block in a
-// macro to "swallow" the semicolon after the macro's use so that it is
-// used like a function without warnings about extra semicolons. It is
-// generally implemented by wrapping the block as a do/while with a false
-// condition so that it executes exactly once. (See, for example,
-// https://gcc.gnu.org/onlinedocs/cpp/Swallowing-the-Semicolon.html)
-// However, some compilers might complain about this, too. For example, you
-// might get a complaint about a loop with a constant expression. These
-// macros take care of these conditions.
+  // Define a pair of macros, VISKORES_SWALLOW_SEMICOLON_PRE_BLOCK and
+  // VISKORES_SWALLOW_SEMICOLON_POST_BLOCK that can be used around a block in a
+  // macro to "swallow" the semicolon after the macro's use so that it is
+  // used like a function without warnings about extra semicolons. It is
+  // generally implemented by wrapping the block as a do/while with a false
+  // condition so that it executes exactly once. (See, for example,
+  // https://gcc.gnu.org/onlinedocs/cpp/Swallowing-the-Semicolon.html)
+  // However, some compilers might complain about this, too. For example, you
+  // might get a complaint about a loop with a constant expression. These
+  // macros take care of these conditions.
```

Comment:
leading space in the comments

Reply:
I must have activated some formatting in my IDE that changed a bunch of indentation. Backed out of those changes so only the lines with actual changes are modified.